samples/snippets/csharp/concepts/linq/LinqSamples.Test/BasicsTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Xunit;
+using static LinqSamples.Test.Shared;
+
+namespace LinqSamples.Test;
+
+public class BasicsTest
+{
+    [Fact]
```

Comment:
I only wrote unit tests for samples which had output in the comments, because (presumably) the output was important to demonstrate the point of the sample; and I tested specifically for that. (One issue: because some of the samples explicitly output `\r\n` as the newline, the tests might fail on a Linux/Mac system.) If the tests were run as part of CI, it would ensure these output comments remain up to date.

If there was no such output in the comments (and if the output wasn't referenced in the text), I didn't think there was anything that could be tested (assuming the sample builds as part of a larger project and runs without exception.) Without such output, I think the concrete results of the sample -- returns 5 `Student` objects whose names are "X" -- are irrelevant.